src/src/com/microsoft/aad/adal/DeveloperAuthenticationException.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.aad.adal;
```

Comment:
We need to have the copy right block in the file. see other existing .java files for reference.